wiki/Building-From-Source.md

```diff
@@ -43,7 +43,7 @@ If you are on Linux, install these additional packages:
   
   ```bash
   sudo dnf groupinstall 'Development Tools' | For c++ and build tools
-  sudo dnf install yasm libdrm-devel vulkan-headers pipewire-jack-audio-connection-kit-devel atk-devel gdk-pixbuf2-devel cairo-devel rust-gdk0.15-devel x264-devel vulkan-devel libunwind-devel clang openssl-devel alsa-lib-devel libva-devel
+  sudo dnf install nasm yasm libdrm-devel vulkan-headers pipewire-jack-audio-connection-kit-devel atk-devel gdk-pixbuf2-devel cairo-devel rust-gdk0.15-devel x264-devel vulkan-devel libunwind-devel clang openssl-devel alsa-lib-devel libva-devel
```

Comment:
Smol correction, but you probably don't need `x264-devel` for compiling anymore since it downloads it